pom.xml

```
@@ -66,12 +66,12 @@
     <dependency>
       <groupId>uk.gov.ons.ssdc</groupId>
       <artifactId>ssdc-rm-common-entity-model</artifactId>
-      <version>4.18.0</version>
+      <version>4.19.0-SNAPSHOT</version>
```

I think snapshot should be removed?

```suggestion
      <version>4.19.0</version>
```

---

Yeah so it will be removed but since you're building it locally, it'll have the snapshot in the version name. I'll update this everywhere once I've merged in the common entity changes